implementations/jmc/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu
+LABEL description="JSON Model Compiler Setup for Benchmarking"
+
+RUN apt update
+RUN apt upgrade -y
+
+# build & utils
+RUN apt install -y --no-install-recommends --fix-missing \
+        vim git ca-certificates libtool m4 time automake make pkgconf gcc g++ texinfo
+
+# libs and all
+RUN apt update
+RUN apt install -y --no-install-recommends --fix-missing \
+        libpcre2-dev libre2-dev libjansson-dev python3 python-is-python3 python3-venv python3-pip
+
+RUN apt clean
```

Comment:
Is there any reason all this can't be combined into a single line? See for example the Dockerfile for `ajv-bun`. This would just limit the space taken by intermediate layers.